src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py

```diff
@@ -166,35 +157,30 @@ def create_build(location,
         except FileNotFoundError as e:
             logger.warning("Unable to find file %s", str(e))
 
-def acr_multi_build_definition_create(registry_name,
-                                      repository,
-                                      tags=None,
+def acr_multi_build_definition_create(registry,
+                                      images=None,
                                       multi_arch_tag=None,
                                       build_name=None,
                                       resource_group_name=None,
                                       username=None,
                                       password=None,
                                       yaml=None):
-    if yaml and not build_name:
-        raise CLIError("Please specify a build-name")
-    if not yaml and (not multi_arch_tag or not tags):
+    if not yaml and (not multi_arch_tag or not images):
         raise CLIError("Please specify the multiArch-tag and the tags that form the multi-architecture flux")
-    if not build_name:
-        build_name = repository.replace("/", ";") + ":" + multi_arch_tag
-    return _validate_user_credentials(registry_name,
-                                      repository,
+
+    return _validate_user_credentials(registry,
```

Comment:
Revert `registry` back to `registry_name` and register it similar to https://github.com/AppPlatPerf/azure-cli/blob/master/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L49